tests/acceptance/features/webUITrashbin/trashbinFilter.feature

```diff
@@ -0,0 +1,70 @@
+Feature: Filter files/folders
+
+  As a user
+  I would like to filter files/folders
+  So that I can make the file/folder list more clear
+
+  Background:
+    Given user "user1" has been created with default attributes
+    And user "user1" has logged in using the webUI
+    And the user has created file ".simpleHiddenFile"
+    And the user has created folder ".simpleHiddenFolder"
+    And the following files have been deleted by user "user1"
+      | name                |
+      | lorem.txt           |
+      | simple-folder       |
+      | .simpleHiddenFile   |
+      | .simpleHiddenFolder |
+    And the user has browsed to the trashbin page
+
+  Scenario: user filters files and folders using keyword when the files filter, folders filter, and hidden filter are enabled
+    When the user enables file filter using the webUI
+    And the user enables folder filter using the webUI
+    And the user enables the setting to view hidden files on the webUI
+    And the user filters the file list by "simple" on the webUI
+    Then as "user1" all files and folders containing pattern "simple" in their name should be listed in the trash on the webUI
```

Comment:
Why do you need to send "user1" here. Since the user is already logged and the trashbin elements are obtained from the webUI.

Reply:
Same for all other steps that applies to.

Reply:
The trashbin elements are obtained from  the api and are compared with the elements listed in the webUI

Reply:
i fixed this